app/components/show/access_rights_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= access_rights %>
+<%= access_rights.join(', ') %>
```

Comment:
how about using `.to_sentence`? That's what the index page is using.

Reply:
fixed

Reply:
<img width="437" alt="Screen Shot 2022-02-25 at 9 49 28 AM" src="https://user-images.githubusercontent.com/7741604/155805510-c2225505-f9f7-46b4-ac9a-20500bbb6566.png">